doc/tutorial.md

```diff
@@ -970,7 +1000,7 @@ It should note that the redirect forwarding may harm performance to a certain de
 
 ## Bonding Device
 
-For Bonding device, both `DPVS` and connected Switch/Router need to set the Bonding interfaces with *same* Bonding mode. Note `DPVS` just supports bonding mode 0 and 4 for now. To enable Bonding device on `DPVS`, pls refer `conf/dpvs.bond.conf.sample`. Each Bonding device needs one or more DPDK Physical device (`dpdk0`, ...) to work as it's slaves.
+For Bonding device, both `DPVS` and connected Switch/Router need to set the Bonding interfaces with *same* Bonding mode. Note `DPVS` just supports bonding mode 0 and 4 for now. To enable Bonding device on `DPVS`, pls refer [conf/dpvs.bond.conf.sample](../conf/dpvs.bond.conf.sample). Each Bonding device needs one or more DPDK Physical devices (`dpdk0`, ...) to work as it's slaves.
```

Comment:
Note **that** `DPVS` 
To enable Bonding device on `DPVS`, **please refer to** [conf/dpvs.bond.conf.sample](../conf/dpvs.bond.conf.sample). Each Bonding device needs one or more DPDK Physical devices (`dpdk0`, ...) to work as **its** slaves.